From: Gordon Ball <gordon@chronitis.net>
Date: Thu, 2 Nov 2017 11:00:10 +0000
Subject: Disable sphinxcontrib_github_alt

This extension is not packaged at this time (although is only a single
file which could perhaps be bundled).
---
 docs/conf.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 83ac00c..ec529ae 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -28,7 +28,6 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
-    'sphinxcontrib_github_alt',
 ]
 
 github_project_url = "https://github.com/jupyter/jupyter_core"
